http/http-api/src/test/groovy/com/okta/commons/http/QueryStringTest.groovy

```diff
@@ -90,4 +90,13 @@ class QueryStringTest {
         def qs = new QueryString(query)
         assertEquals qs.toString(), "key1=value1&key2=value2"
     }
+
+    @Test
+    void testWithDateObjectAsValue() {
+        Date sinceDate = new Date("Thu Nov 30 13:15:16 PST 2017")
+        def query = [since:sinceDate]
+        def qs = new QueryString(query)
+        assertEquals qs.toString(), "since=2017-11-30T21%3A15%3A16Z"
+        assertEquals qs.toString(), "since=" + URLEncoder.encode("2017-11-30T21:15:16Z", "UTF-8")
```

Comment:
nit: hardcode the encoding here similar to the previous line

Reply:
Not sure if I got what you meant. Line #100 was coded to indicate clearly that line #99 is an encoded variant of the content in line #100. If its confusing, I can remove #100 in its totality?

Reply:
oh, i see, yes, plase do

Reply:
done